code/datums/datum.dm

```diff
@@ -49,6 +49,9 @@
 	/// A weak reference to another datum
 	var/datum/weakref/weak_reference
 
+	///Lazy associative list of currently active cooldowns.

```

Comment:
Can u include the assoc key and value that'll be in this list?

Reply:
Something like cooldown index -> return of add_timer (usually the truthy value of -1 unless someone makes a stoppable timer version)?

Reply:
i ment in the comment

Reply:
I know, but I don't understand what you are asking.
Maybe give an example?

Reply:
add the structure of the list in the code comment so people looking at it know what the list contains without having to go hunting for it.

Reply:
Like this?

Reply:
yes perfect